src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -325,24 +326,31 @@ class %s extends {$options['class']}
 }
 
 require $autoloadFile;
+if (\PHP_VERSION_ID >= 80100 || \PHP_VERSION_ID < 70400) {
```

Comment:
`\PHP_VERSION_ID < 70400` is useless here. the whole preloading feature is requiring PHP 7.4+ (as PHP 7.3 does not have preloading at all in PHP)